streams-bootstrap-core/src/test/java/com/bakdata/kafka/StreamsBuilderXTest.java

```diff
@@ -523,7 +524,11 @@ public void buildTopology(final StreamsBuilderX builder) {
                 final KStreamX<String, String> input =
                         builder.stream("input", ConsumedX.with(Serdes.String(), Serdes.String()));
                 final GlobalKTable<String, String> otherInput =
-                        builder.globalTable("table_input", MaterializedX.with(Serdes.String(), Serdes.String()));
+                        builder.globalTable("table_input",
+                                // Kafka 4.0 likely contains a bug if Materialized does not have a name
+                                MaterializedX.<String, String, KeyValueStore<Bytes, byte[]>>as("store")
+                                        .withKeySerde(Serdes.String())
+                                        .withValueSerde(Serdes.String()));
```

Comment:
Should we split this into two tests then and disable the one with the unnamed store for now to keep better track of this?